platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/test/HappyPathTest.java

```diff
@@ -10,20 +10,21 @@
 import org.hiero.otter.fixtures.TimeManager;
 import org.hiero.otter.fixtures.Validator.LogFilter;
 import org.hiero.otter.fixtures.Validator.Profile;
+import org.hiero.otter.fixtures.turtle.TurtleNetwork;
 import org.junit.jupiter.api.Disabled;
 
 public class HappyPathTest {
 
     @Disabled
     @OtterTest
     void testHappyPath(TestEnvironment env) throws InterruptedException {
+        env.start();
         final Network network = env.network();
         final TimeManager timeManager = env.timeManager();
 
         // Setup simulation
         network.addNodes(4);
         network.start(Duration.ofMinutes(1L));
-        env.generator().start();
```

Comment:
I do not think that this is a good idea. The `TransactionGenerator` either must be controlled by the user, or we can bind it to the lifecycle of the network/node. But sending transactions while the network has not been set up seems wrong.

Reply:
It's now bound to the lifecycle.